packages/runner/sync-gmail-raw/service/email_service.go

```diff
@@ -148,14 +148,19 @@ func (s *emailService) ReadEmailFromGoogle(gmailService *gmail.Service, username
 func (s *emailService) ReadEmailsForUsername(gmailService *gmail.Service, gmailImportState *entity.UserGmailImportState) (*entity.UserGmailImportState, error) {
 	countEmailsExists := int64(0)
 
-	userMessages, err := gmailService.Users.Messages.List(gmailImportState.Username).MaxResults(s.cfg.SyncData.BatchSize).PageToken(gmailImportState.Cursor).Do()
+	userInboxMessages, err := gmailService.Users.Messages.List(gmailImportState.Username).Q("in:inbox").MaxResults(s.cfg.SyncData.BatchSize).PageToken(gmailImportState.Cursor).Do()
```

Comment:
if the user has skipped the inbox for an email, we're not going to receive it. this is ok for skipping drafts but will skip all the other folders

Reply:
for example:
i have a Github label. all my emails have a rule to skip the inbox and move to that folder, so all the emails from github will have just the "github" label

Reply:
true, thanks for that, I will try a new approach to tackle what you mentioned